integration-tests/integration/prisonerProfile/prisonerWorkAndSkills.cy.js

```diff
@@ -559,9 +549,8 @@ context('Prisoner Work and Skills', () => {
           cy.get($workScheduleLink).contains('View 7 day schedule')
         })
 
-        cy.get('[data-test="work-unacceptableAbsence"]').contains('Last 30 days')
-        cy.get('[data-test="work-unacceptableAbsence"]').contains('4')
-        cy.get('[data-test="work-absencesLink"]').contains('View unacceptable absences for the last 6 months')
+        cy.get('[data-test="work-unacceptableAbsence"]').should('not.exist')
+        cy.get('[data-test="work-absencesLink"]').should('not.exist')
```

Comment:
Some bits of the work and skills were conditional on a list of 'accelerator' caseloads containing the user's active caseload - now the only way the user can access this is if they have NO caseload, so these bits are no longer visible at all